apps/cowswap-frontend/src/modules/advancedOrders/hooks/useAdvancedOrdersActions.ts

```diff
@@ -21,25 +20,16 @@ export function useAdvancedOrdersActions() {
 
   const naviageOnCurrencySelection = useNavigateOnCurrencySelection()
   const updateCurrencyAmount = useUpdateCurrencyAmount()
-  const resetTradeQuote = useResetTradeQuote()
   const cowAnalytics = useCowAnalytics()
   const debouncedTradeAmountAnalytics = useMemo(() => createDebouncedTradeAmountAnalytics(cowAnalytics), [cowAnalytics])
 
   const updateAdvancedOrdersState = useUpdateAdvancedOrdersRawState()
 
   const onCurrencySelection = useCallback(
     (field: Field, currency: Currency | null) => {
-      // Reset the output field until we fetch quote for new selected token
-      // This is to avoid displaying wrong amounts in output field
```

Comment:
This was an execessive amount reset which caused sell amount resetting to 1 when you change sell token